src/js/components/SelectMultiple/stories/Variant.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+
+import { Box, SelectMultiple, Text } from 'grommet';
+
+const dummyOptions = [
+  'Apple',
+  'Orange',
+  'Banana',
+  'Grape',
+  'Melon',
+  'Strawberry',
+  'Kiwi',
+  'Mango',
+  'Raspberry',
+  'Rhubarb',
+];
+
+export const Variant = () => {
```

Comment:
Instead of "Variant" can we call this story "showSelectedInline"

Reply:
Yup, done